Sources/SwiftFormats/FormatStyle+CoreGraphics.swift

```diff
@@ -44,6 +44,32 @@ public struct CGPointParseStrategy: ParseStrategy {
 
 // MARK: -
 
+public extension CGPoint {
+    init(_ string: String) throws {
+        self = try CGPointFormatStyle().parseStrategy.parse(string)
+    }
+
+    init<T, Value>(_ value: Value, format: T) throws where T: ParseableFormatStyle, Value: StringProtocol, T.Strategy == CGPointParseStrategy {
```

Comment:
Maybe a better signature? (Using more explicit names and following ParseableStrategy's naming… (I also need to follow this advice through more of the code - code is due a naming refactor))

```suggestion
   init<Format, ParseInput>(_ input: ParseInput, format: T) throws where Format: ParseableFormatStyle, ParseInput: any StringProtocol, Format.Strategy == CGPointParseStrategy {
```

Reply:
Good call. These signatures are already long and complicated, might as well do our future selves and any other contributors a favour and make it readable 😆